event_model/tests/test_em.py

```diff
@@ -1116,7 +1189,7 @@ def all_factory(name, doc):
         return [header_collector], []
 
     rr = event_model.RunRouter([all_factory])
-    with pytest.warns(UserWarning, match='1.14.0'):
+    with pytest.warns(UserWarning, match='1.14.0'), pytest.raises(LocalException3):
```

Comment:
I had no idea you could chain contexts like this!

Reply:
It works with files so I took a leap.